packages/common/src/utils.ts

```diff
@@ -0,0 +1,5 @@
+import type BigNumber from 'bignumber.js'
+
+export function weiToEth(wei: BigNumber): string {
+    return wei.div(1e18).toString()
```

Comment:
```suggestion
    return wei.div(1e18).toString(10)
```
We should put a radix of `10` here to ensure decimal formatting and not exponential notation.

Reply:
Thanks. I was thinking that was the default. 
`If the radix is not specified, the preferred radix is assumed to be 10.`